src/Personnummer.php

```diff
@@ -451,6 +459,20 @@ private function validateNumPartsForVgr(): bool
         return false;
     }
 
+    /**
+     * Make sure the the issuance date of the sll-reserve number is not greater than the current year + 1.
+     * 
+     * @return bool
+     */
+    private function validateNumPartsForSll(): bool
+    {
+        $parts = $this->parts;
+        $now = new DateTime();
+
+        return (int)($parts['fullYear']) <= $now->format('Y') + 1
+                && (int)($parts['fullYear']) > 1870;
```

Comment:
Prevents sll-reserve number issuance years that are from the future and issuance years older 1870.